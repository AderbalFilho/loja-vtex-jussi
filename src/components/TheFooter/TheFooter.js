function TheFooter() {
  return null;
}

export default TheFooter;
