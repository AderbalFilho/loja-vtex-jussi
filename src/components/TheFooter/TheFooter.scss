.the-footer {
  background-color: var(--black);
  bottom: 0;
  left: 0;
  padding: 24px;
  position: fixed;
  width: 100%;
}

.the-footer__container {
  align-items: center;
  display: flex;
  justify-content: space-between;
  max-width: 1024px;
  margin: 0 auto;
}

.the-footer__left {
  max-height: 27px;
  max-width: 142px;
}

.the-footer__right {
  display: flex;
}

.the-footer__icon-link {
  height: 32px;
  width: 32px;

  &:not(:first-child) {
    margin-left: 17px;
  }
}

.the-footer__icon {
  height: 100%;
  width: 100%;
}
