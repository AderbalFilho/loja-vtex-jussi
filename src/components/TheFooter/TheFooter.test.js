import React from 'react';
import TestRenderer from 'react-test-renderer';

import TheFooter from './TheFooter';

describe('The Header', () => {
  it('matches snapshot', () => {
    const testRenderer = TestRenderer.create(<TheFooter />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
