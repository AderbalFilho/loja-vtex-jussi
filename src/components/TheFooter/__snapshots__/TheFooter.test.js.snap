// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header matches snapshot 1`] = `
<footer
  className="the-footer"
>
  <div
    className="the-footer__container"
  >
    <div
      className="the-footer__left"
    >
      <img
        alt="WPP Company logo"
        src=""
        srcSet=",  2x"
      />
    </div>
    <div
      className="the-footer__right"
    >
      <a
        className="j-link the-footer__icon-link"
        href="https://pt-br.facebook.com/agencia.jussi/"
        rel="noreferrer noopener"
        target="_blank"
      >
        <img
          alt="Facebook logo"
          className="the-footer__icon"
          src=""
          srcSet=",  2x"
        />
      </a>
      <a
        className="j-link the-footer__icon-link"
        href="https://www.instagram.com/jussi/?hl=pt-br"
        rel="noreferrer noopener"
        target="_blank"
      >
        <img
          alt="Instagram logo"
          className="the-footer__icon"
          src=""
          srcSet=",  2x"
        />
      </a>
      <a
        className="j-link the-footer__icon-link"
        href="https://br.linkedin.com/company/agencia-jussi"
        rel="noreferrer noopener"
        target="_blank"
      >
        <img
          alt="Linkedin logo"
          className="the-footer__icon"
          src=""
          srcSet=",  2x"
        />
      </a>
    </div>
  </div>
</footer>
`;
